async-openai/src/audio.rs

```diff
@@ -46,4 +49,17 @@ impl<'c, C: Config> Audio<'c, C> {
 
         Ok(CreateSpeechResponse { bytes })
     }
+
+    /// Generates audio from the input text.
+    pub async fn speech_stream(
+        &self,
+        request: CreateSpeechRequest,
+    ) -> Result<SpeechResponseStream, OpenAIError> {
+
+        let stream = Box::pin(self.client.post_raw_stream("/audio/speech", request)
```

Comment:
Where is the `stream=True` provided? It seems that is equivalent to non-streaming in absence of the `stream=True`

Reply:
The `stream` parameter does not do anything from my tests - the response is streamed + chunked regardless. You can confirm it using curl / postman or something else. I also have a short example that playbacks the audio on the fly instead of writing it to disk - and that typically starts straightaway, I can add it to the example folder if that is helpful - I felt like it was slightly out of scope of the library but I don't mind adding it if that's helpful.